components/clm/tools/clm4_5/OLMT/runcase.py

```diff
@@ -0,0 +1,1591 @@
+#!/usr/bin/env python
```

Comment:
There's a lot of files in this directory that look like they just have additions (new files?). Is it possible that this was a submodule that accidentally had the actual source copied over and committed to the repo?

Reply:
Are you referring to clm4_5 and deeper? These files exist in the E3SM repo as regular files; see, e.g., https://github.com/E3SM-Project/E3SM/tree/master/components/clm/tools/clm4_5. Plus these are all mod'ed files, not new ones, unless I'm missing something.

Reply:
Sorry was referring to the OLMT directory. Some of these looked like new files based on the git diffs. Looks like these exist in the E3SM repo though, so I retract my comment.